test/box-luatest/gh_7149_forbid_ddl_until_box_schema_upgrade_test.lua

```diff
@@ -21,62 +21,3 @@ g.test_schema_access = function()
         box.schema.upgrade()
     end)
 end
-
```

Comment:
This test now looks pointless. Let's drop it altogether?

Reply:
Let's keep it, there is a check for the another minor issue.